docs/en/02_Developer_Guides/01_Templates/11_Partial_Template_Caching.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Partial Template Caching
+summary: Cache a section of a template Reduce rendering time with cached templates and understand the limitations of the ViewableData object caching.
+icon: tags
+---
+
+## Partial template caching
+
+Partial template caching is a feature that allows caching of rendered portions of templates. Cached content
+is fetched from a [cache backend](../performance/caching), instead of being regenerated repeatedly.
+
+
+### Base syntax
+
+Here is an example of the basic usage. This is not a definitive example of the syntax, but it shows the most
+common use case.
+
+[note]
+See also [Complete Syntax definition](#complete-syntax-defintition) section
+[/note]
+
+```ss
+<% cached $CacheKey if $Condition %>
+  <!-- Content block -->
+<% end_cached %>
+```
+
+The key parts are `$CacheKey`, `$Condition` and `<!-- Content block -->`.  
+The following sections explain every one of them in more detail.
+
+
+#### $CacheKey
+
+The `$CacheKey` is a string identifying the block content within the cache storage.
+
+The form is `[$key[, $...]]`, where `$key` is any valid template expression and you can have any number of them delimited by comma.
+
+Ultimate value of the cache key includes the following items:
+  - hash of `SilverStripe\View\SSViewer::$global_key` rendered within current template context
+  - hash of unrendered `<!-- Content block -->`
```

Comment:
in order to render the has of the Content block, you'd need to actually render it every time, including all of the required database quereies.  This totally defeats the purpose of template caching =)

Reply:
Fair enough, needs clarification.
This is a hash of **unrendered** <!-- Content block -->. That does not require any rendering and **calculated at the template compilation stage**.

Reply:
... unless it has DataObjects nested inside it?

Reply:
Not sure what you mean by that. DataObjects are irrelevant in terms of the template syntactic markup.

Reply:
yeah but how do you calculate the hash of `<!-- Content block -->`?

Seems like you need to render the whole thing to get the hash?

I don't understand how that would make a good cache key component since the whole point of the cache is to NOT have to render the whole thing?